web/src/views/Apply/FormQuestions.tsx

```diff
@@ -9,26 +9,29 @@ export const FormQuestions = () => {
     firstName: Yup.string().required('Debes incluir tu nombre'),
     lastName: Yup.string().required('Debes incluir tu apellido'),
     email: Yup.string().email('Correo inválido').required('Debes incluir tu email'),
-    institution:  Yup.string().required('Debes incluir tu centro de estudios'),
+    institution: Yup.string().required('Debes incluir tu centro de estudios'),
     links: Yup.array()
       .of(
         Yup.object().shape({
-          url: Yup.string().required('Debes incluir tu link personal'),
+          url: Yup.string()
+            .url('Debes incluir un link válido')
+            .required('Debes incluir tu link personal'),
```

Comment:
@dguzt Should we keep the personal links as required? I think it should be optional. On the other hand, the form does not give us a lot of information about the candidate without this field. 

What do you think about adding some additional questions?

Reply:
@dguzt  x2  xd

Reply:
This question could be optional, but we can motivate the applicant to fill it to accelerate his/her apply process. We can use the following text as question description:
- "Puedes incluir uno o varios links personales que desees mostrarnos (GitHub, LinkedIn, página web personal, etc). Nos ayudarás mucho en tu proceso de postulación."

Text can change (talk to Oliver to check the ux writing).

Btw, the code link is required, but it's in the next question. Ok with that.